src/server/server.js

```diff
@@ -39,6 +39,10 @@ app.get('/orient/generate', (req, res) => {
   generateDatabase(res);
 });
 
+app.get('/tweet/quotedby/:id', (req, res) => {
+  getQuotedTweetFromParent(req, res, req.params.id);
```

Comment:
any reason you're passing in the req and the params id then using req.params.id in getQuotedTweetFromParent